cmake/core-files.cmake

```diff
@@ -630,6 +639,7 @@ set(MBGL_CORE_FILES
     include/mbgl/util/tileset.hpp
     include/mbgl/util/timer.hpp
     include/mbgl/util/traits.hpp
+    include/mbgl/util/tuple.hpp
```

Comment:
I'm not sure why this was added 🤔

Reply:
This file was added in https://github.com/mapbox/mapbox-gl-native/pull/10832, but it looks like the `scripts/generate-core-files.sh` wasn't run.